standup-irc.js

```diff
@@ -432,15 +432,8 @@ var commands = {
                 let response = api.status.create(user, project, status);
 
                 response.once('ok', function(data) {
-                    let msg;
                     let user_url = `${config.standup.url}/user/${user}/`;
-
-                    if (Math.random() < 0.97) {
-                        msg = 'Ok, submitted ';
-                    } else {
-                        msg = 'Okeeday, meesa submittin ';
-                    }
-                    msg += 'status #' + data.id + ' for ' + user_url;
+                    let msg = `Ok, submitted #${data.id} for ${user_url}`;
```

Comment:
Can you change this to:

```
let msg = 'Ok, submitted ' + data.id + ' for ' + user_url;
```

Then it follows the style of the rest of the file.

At some point, if you want to change the style for the whole file, you should do that in a new PR and also test it out on heroku to make sure it works in the environment it's running in.